tripy/tests/integration/test_reshape.py

```diff
@@ -86,12 +86,16 @@ def test_flatten_invalid_dims(self):
             tp.flatten(a, start_dim=2, end_dim=1)
 
     def test_flatten_single_dim(self):
-        shape = (2, 3, 4)
-        a = tp.ones(shape)
-        # Flattening a single dimension should not change the output
-        b = tp.flatten(a, start_dim=1, end_dim=1)
-        assert b.shape == (2, 3, 4)
-        assert np.array_equal(cp.from_dlpack(b).get(), np.ones(shape, dtype=np.float32))
+        from tripy.logging import logger
+
+        # This is required to print intermediate data representations.
+        with tp.logger.use_verbosity("ir"):
```

Comment:
Was this only for debugging?

Reply:
Yes, I will remove this once I open up the PR for review.